projects/rescuechi/petfinder/data_cleaner.py

```diff
@@ -0,0 +1,54 @@
+import pandas as pd
+from pathlib import Path
+
+DATA_FOLDER = Path(__file__).parent / "data"
```

Comment:
If I was to try to run this on my own I don't think I'd remember how to update this path to pull the file correctly. Could you add a comment explaining that this DATA_FOLDER command should be edited to wherever the pkl file was saved on the local machine? I think you mentioned that when we paired.

Reply:
I ended up making a separate PR for all of this data_clear code, and removed it from this PR 😄 I additionally started a new PR with some loose ends, plus a new-and-improved README that explains some of the file locations and such!